From 4506f19d58fa08310bdc397730e78d2efdebfd68 Mon Sep 17 00:00:00 2001
From: Jingyi Wang <quic_jingyw@quicinc.com>
Date: Fri, 6 Dec 2024 14:41:13 +0800
Subject: [PATCH] FROMLIST: arm64: dts: qcom: qcs8300: Add capacity and DPC
 properties

The "capacity-dmips-mhz" and "dynamic-power-coefficient" are used to
build Energy Model which in turn is used by EAS to take placement
decisions. So add it to QCS8300 SoC.

Change-Id: I718d2d39f7261d5696919b97a8709964060b485b
Signed-off-by: Jingyi Wang <quic_jingyw@quicinc.com>
Reviewed-by: Konrad Dybcio <konrad.dybcio@oss.qualcomm.com>
Upstream-Status: Submitted [https://lore.kernel.org/all/20241206-qcs8300_dpc-v1-1-af2e8e6d3da9@quicinc.com]
---
 arch/arm64/boot/dts/qcom/qcs8300.dtsi | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcs8300.dtsi b/arch/arm64/boot/dts/qcom/qcs8300.dtsi
index f7b5e67f939e..06dd015b1757 100644
--- a/arch/arm64/boot/dts/qcom/qcs8300.dtsi
+++ b/arch/arm64/boot/dts/qcom/qcs8300.dtsi
@@ -31,6 +31,8 @@
 			next-level-cache = <&l2_0>;
 			power-domains = <&cpu_pd0>;
 			power-domain-names = "psci";
+			capacity-dmips-mhz = <1946>;
+			dynamic-power-coefficient = <472>;
 
 			l2_0: l2-cache {
 				compatible = "cache";
@@ -48,6 +50,8 @@
 			next-level-cache = <&l2_1>;
 			power-domains = <&cpu_pd1>;
 			power-domain-names = "psci";
+			capacity-dmips-mhz = <1946>;
+			dynamic-power-coefficient = <472>;
 
 			l2_1: l2-cache {
 				compatible = "cache";
@@ -65,6 +69,8 @@
 			next-level-cache = <&l2_2>;
 			power-domains = <&cpu_pd2>;
 			power-domain-names = "psci";
+			capacity-dmips-mhz = <1946>;
+			dynamic-power-coefficient = <507>;
 
 			l2_2: l2-cache {
 				compatible = "cache";
@@ -82,6 +88,8 @@
 			next-level-cache = <&l2_3>;
 			power-domains = <&cpu_pd3>;
 			power-domain-names = "psci";
+			capacity-dmips-mhz = <1946>;
+			dynamic-power-coefficient = <507>;
 
 			l2_3: l2-cache {
 				compatible = "cache";
@@ -99,6 +107,8 @@
 			next-level-cache = <&l2_4>;
 			power-domains = <&cpu_pd4>;
 			power-domain-names = "psci";
+			capacity-dmips-mhz = <1024>;
+			dynamic-power-coefficient = <100>;
 
 			l2_4: l2-cache {
 				compatible = "cache";
@@ -116,6 +126,8 @@
 			next-level-cache = <&l2_5>;
 			power-domains = <&cpu_pd5>;
 			power-domain-names = "psci";
+			capacity-dmips-mhz = <1024>;
+			dynamic-power-coefficient = <100>;
 
 			l2_5: l2-cache {
 				compatible = "cache";
@@ -133,6 +145,8 @@
 			next-level-cache = <&l2_6>;
 			power-domains = <&cpu_pd6>;
 			power-domain-names = "psci";
+			capacity-dmips-mhz = <1024>;
+			dynamic-power-coefficient = <100>;
 
 			l2_6: l2-cache {
 				compatible = "cache";
@@ -150,6 +164,8 @@
 			next-level-cache = <&l2_7>;
 			power-domains = <&cpu_pd7>;
 			power-domain-names = "psci";
+			capacity-dmips-mhz = <1024>;
+			dynamic-power-coefficient = <100>;
 
 			l2_7: l2-cache {
 				compatible = "cache";
-- 
2.17.1

